### PR TITLE
Light client wallet (read only) for external addresses

### PR DESCRIPTION
**Pull request by g11tech**
**Motivation**
A light client wallet (read only) for ethereum post merge.

This PR extends the light client demo to make it workable for connecting with a kilnv1 post merge devnet. 

~~Some of the functionality like storage is temporarily removed as the demo is not uptodate with the latest light client apis. Also the library use is not optimmal and hacky. Some things further fixing as the SSE end point seems to be erroring, so real time updates are not happening.~~
**Description**
Context: https://github.com/ChainSafe/lodestar/issues/3796

TODOs:
- [x] Review and optimize library use
- [x] Fix SSE endpoind error for realtime update
~~- [ ] Restore the storage class~~ functionality has been dropped for now till demo stabilizes
- [x] Code refac
- [x] rpc endpoint configurability
- [x] Fix tests and other cleanup/prettier
- [x] Display balance in eth!
- [x] RPC error handling

BUT it works @dapplion @wemeetagain !!!
image

thanks to ethereumjs's libraries! :heart:  and not to mention `getProof`! (cc: @ryanio @jochem-brouwer)

### CODE REVIEW

**Comment by g11tech** [comment]
error handling added for el rpc requests:
image

**Comment by dapplion** [comment]
This is really cool thank you so much!

**Comment by dapplion** [comment]
Where should we add CI regression tests to ensure it all works well? Lodestar + EthereumJS -> sync then query both

**Comment by g11tech** [comment]
> Where should we add CI regression tests to ensure it all works well? Lodestar + EthereumJS -> sync then query both

Thinking of adding ethereumjs in the lodestar merge CI and then run,

**Comment by mpetrunic** [comment]
@dapplion @g11tech Let me know when this is ready, we could merge to some branch and create some experimental releases so it's visible in explorer. Let me know what you think?

**Comment by dapplion** [comment]
> @dapplion @g11tech Let me know when this is ready, we could merge to some branch and create some experimental releases so it's visible in explorer. Let me know what you think?

Sounds good to me!

File: src/App.tsx [hunk 1]
```diff
@@ -63,13 +86,40 @@ export default function App(): JSX.Element {
...
+          setAccountReqStatus({result: accountReqStatus.result, loading: `Waiting for an execution payload`});
+          return;
+        } else {
+          setAccountReqStatus({result: accountReqStatus.result, loading: `Fetching status from ${elRpcUrl}`});
+          fetchAndVerifyAddress({web3: web3.current, executionPayload, address})
+            .then((verifiedAccount) => {
+              setAccountReqStatus({result: verifiedAccount});
+            })
+            .catch((e) => {
+              setAccountReqStatus({result: accountReqStatus.result, error: e});
```
Comment (dapplion) [inline src/App.tsx]: Recommended pattern in React async useEffect is to just declare and async function in the useEffect body and call it with a .catch() then you can use await

**Review by dapplion — APPROVED** [review]
Let's go! Excellent stuff @g11tech :fire: 

For me I'll merge and keep iterating on subsequent PRs

**Pull request merged**